packages/helpers/packages/CookieHelpers/index.js

```diff
@@ -32,3 +32,13 @@ export const setCookie = (name, value, expireDays = 90, path = '/') => {
   const expires = `expires=${d.toUTCString()}`;
   document.cookie = `${name}=${value};${expires};path=${path}`;
 };
+
+/**
+ * delete a cookie
+ * @author Reza Erami
+ * @param name
+ */
+export const deleteCookie = (name) => {
+  const expires = `expires=${0}`;
+  document.cookie = `${name}=${null};${expires}`;
```

Comment:
It is better(and safer) to use `01 Jan 1970` to remove cookie: 
https://stackoverflow.com/questions/2144386/how-to-delete-a-cookie